async function handle(err, result){
    if(err){
        console.log("error on aplication");
        return;
    }

}